app/graphql/types/query_type.rb

```diff
@@ -0,0 +1,37 @@
+module Types
```

Comment:
I'll be working in this file this evening, getting the appropriate filters added for dog(s) / user(s).